<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sokar</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="centralWidget">
       <layout class="QVBoxLayout" name="verticalLayout2">
        <item>
         <widget class="Sokar::FileTree" name="fileTree"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>≈Åaduj</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Sokar::DicomTabs" name="dicomTabs" native="true"/>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>F&amp;ile</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_GDCM"/>
    <addaction name="actionAbout_Sokar"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTheme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="actionAbout_GDCM">
   <property name="text">
    <string>About &amp;GDCM</string>
   </property>
  </action>
  <action name="actionAbout_Sokar">
   <property name="text">
    <string>A&amp;bout Sokar</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Sokar Help</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string>Theme</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sokar::DicomTabs</class>
   <extends>QWidget</extends>
   <header>widget/dicomtabs.h</header>
  </customwidget>
  <customwidget>
   <class>Sokar::FileTree</class>
   <extends>QTreeView</extends>
   <header>widget/filetree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
